### DISCUSSION TITLE
Image preview of links

### DISCUSSION THREAD

**Discussion by ghost** [aahnik/tgcf#193] [General]
Text messages with links having image preview are also forwarded when filter files whitelist is given as photo. Only pictures should be forwarded and not messages. Example link
https://www.audible.com/pd/The-Adventures-of-Tom-Sawyer-Audiobook/B002V9ZGZW?source_code=AUDORWS0718179KY7

**Comment by aahnik** [comment]
What is happening is expected. This is not a bug. The message is a "text" as you said. The message does not have a photo at all. Telegram just shows the preview.

What do you want?

You want the preview image to be downloaded from the website and then forwarded?

**Reply by ghost** [reply]
Thank you for looking into the issue. I don't want that message/text  to be forwarded at all. I only want to forward images. Hope I am clear now.

**Reply by aahnik** [reply]
I am sorry, I misunderstood your question. I will look into it.

You are saying that the message is getting forwarded, but you don't want it, right?

**Reply by ghost** [reply]
That is correct.

**Comment by tissole** [comment]
I observed this behavior when testing the filters in #131

> Filter photo forwards only photos, but not photos send as files. This filter also forwards links to websites, YouTube, Mega, Udemy, and TG channels. Really!

I think this could be resolved when regex is implemented. Now when a user whitelist photo, those links are sneaking too :)
